python/triton/runtime/interpreter.py

```diff
@@ -1077,10 +1077,13 @@ def _restore_args_dev(self, args_dev, args_hst, kwargs, kwargs_hst):
                 kwarg_dev.data.copy_(kwarg_hst.to(kwarg_dev.device).data)
 
     def __call__(self, *args_dev, **kwargs):
-        # removes reserved keywords from kwargs
-        kwargs = {k: v for k, v in kwargs.items() if k not in RESERVED_KWS}
```

Comment:
Is `RESERVED_KWS` dead code now?

Reply:
Yes, I made a change locally, forgot to upstream... Thanks for catching